packages/quantic/force-app/main/default/lwc/quanticRecentQueriesList/quanticRecentQueriesList.html

```diff
@@ -14,14 +14,14 @@
         <div class="slds-text-body_small empty-list-message slds-p-horizontal_medium slds-var-m-vertical_small">{labels.emptyListLabel}</div>
       </template>
       <ul>
-        <template for:each={queries} for:item="query">
-          <li class="slds-var-m-vertical_x-small slds-grid" key={query}>
-            <div class="slds-col recent-query__container">
+        <template for:each={queries} for:item="query" for:index="index">
+          <li key={query} class="slds-var-m-vertical_x-small slds-grid" value={index} onclick={executeQuery}>
+            <a class="slds-col recent-query__container">
               <lightning-icon class="recent-query__icon slds-current-color" icon-name="utility:search" alternative-text="search icon" size="xx-small"></lightning-icon>
-              <div class="slds-text-body_small nowrap query-text__container">
-                <p class="slds-truncate">{query}</p>
-              </div>
-            </div>
+              <span class="slds-text-body_small nowrap query-text__container slds-truncate slds-text-link">
```

Comment:
Have you tested that the `slds-truncate` works well on this span element as well?

Reply:
yes

Reply:
nice 👍